### ISSUE TITLE
Cannot see in list if calendar is shared (visible in modal only)

### ISSUE THREAD

**Issue by raimund-schluessler** [nextcloud/calendar#4869]
### Steps to reproduce

1. Share a calendar
2. Look at shared calendar in the calendar list on the left

### Expected behavior

There should be an indicator that the calendar is shared.

### Actual behaviour

There is no indicator that the calendar is shared and you have to open the calendar settings modal to figure out that the calendar is shared.
Similar to when a calendar is shared with you (where the avatar of the owner is show), a sharing icon should be visible that indicates that the calendar is shared with someone. Clicking the icon should open the calendar settings / sharing modal. It was like this before Calendar 4.2.0.

See here, the "Default calendar" is shared but you have to open the modal to see this, there is no icon anymore:
<img width="686" alt="shared_calendar" src="https://user-images.githubusercontent.com/2496460/210210188-ec0f7208-1b34-460a-8ae2-aefbaf9d2bf9.png">

### Calendar app version

4.2.0

Related to https://github.com/nextcloud/calendar/pull/4515

**Comment by miaulalala** [comment]
cc @nextcloud/designers

**Comment by privatemaker** [comment]
@nimishavijay @jancborchardt what do you think of the solution I propose in #5083 which I feel addresses OP's issue here?

**Comment by stefan-niedermann** [comment]
cc @nextcloud/designers
